.github/workflows/backport_manual.yml

```diff
@@ -0,0 +1,54 @@
+name: Backport rook/rook upstream cronjob 2
+
+on:
+  push:
+    branches:
+      feature/ksd-72
```

Comment:
What's this file?

Reply:
@ideepika could you check if you can remove the file? I seem to be unable to easily remove it without having to rebase several commits.